src/cmdlib.sh

```diff
@@ -102,3 +113,81 @@ runcompose() {
          ${TREECOMPOSE_FLAGS:-} ${manifest} "$@"
     set +x
 }
+
+prepare_vm() {
+    # rpms to create appliance VM out of
+    rpms=' bash vim-minimal coreutils util-linux procps-ng kmod kernel-modules'
+    rpms+=' cifs-utils' # for samba
+    rpms+=' systemd' # for clean reboot
+    rpms+=' dhcp-client bind-export-libs iproute' # networking
+    rpms+=' rpm-ostree distribution-gpg-keys' # to run the compose
+    rpms+=' selinux-policy selinux-policy-targeted policycoreutils' #selinux
```

Comment:
Maybe nicer to maintain this as a `supermin.txt` file or so in our source?

One thing to think about too is that supermin also supports `--build` explicitly which we could do as part of our container build.  (The whole libguestfs/supermin was designed around pulling content dynamically from the *host*, but in our case the container is static)

Reply:
> Maybe nicer to maintain this as a `supermin.txt` file or so in our source?

i.e. just the rpm requirements in a separate file? 

> 
> One thing to think about too is that supermin also supports `--build` explicitly which we could do as part of our container build. (The whole libguestfs/supermin was designed around pulling content dynamically from the _host_, but in our case the container is static)

yeah I do a --build below, right? I'd rather not do the `--build` as part of the container build because it will just add a rootfs filesystem to our container and will make it larger to download with not real benefit. that's why I put it as part of the `coreos-assembler init`

Reply:
Well, the benefit is it's faster to run each time.

Note that `coreos-assembler init` is intended to be run only once.  If the VM image is cached there, then when you pull a new container, it will be out of date.

We have `cache/`...but really every time the container changes the VM image is going to need to change anyways.  And with this, the container doesn't *do* much without the VM image so...we're trading off download/disk space versus CPU usage every update.